### ISSUE TITLE
Clicking on the Figure 1 or Table 1 in the text, do not take us to figure or table resp.

### ISSUE THREAD

**Issue by varshilmehta** [Vitaliy-1/oldGregg#115]
Clicking on the Figure 1 or Table 1 in the text, do not take us to figure or table resp.

**Comment by Vitaliy-1** [comment]
Yep, this comes from a JATS Parser Plugin. Most probably I'll fix this in the next release. With the new version, it would be possible to use this plugin with any theme, just like it's now only for Old Gregg - display JATS on the article landing page. But modifications are huge, thus it will take some time.
